modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ModalDeleteObjectDefinition.tsx

```diff
@@ -82,7 +82,7 @@ export function ModalDeleteObjectDefinition({
 						'confirm-object-definition-name'
 					)}
 					title={Liferay.Language.get('delete-object-definition')}
-					token={objectDefinition?.name}
+					token={objectDefinition ? objectDefinition.name : ''}
```

Comment:
Hi @Artartur. Good catch! Another way to do that would be  `objectDefinition?.name ?? ''`. But you can keep the way you did it. Thanks!